### ISSUE TITLE
[建议] 加入 GPEN 和 GFPGAN 人脸增强

### ISSUE THREAD

**Issue by DoubleGuo** [AaronFeng753/Waifu2x-Extension-GUI#328]
**建议描述**
建议加入 GPEN 和 GFPGAN 人脸增强
个人感觉GFPgan有点锐化过度
https://github.com/TencentARC/GFPGAN
https://github.com/yangxy/GPEN

**您设想的解决方案**

**类似的实现**
Real-ESRGAN 里就集成了 GFPGAN 可以参考实现
https://github.com/xinntao/Real-ESRGAN

**补充内容**

**Comment by arthur-st** [comment]
@AaronFeng753 Since #535 is marked as completed, does this work now? I can't find it in the application, using the premium version.

**Comment by AaronFeng753** [comment]
> @AaronFeng753 Since #535 is marked as completed, does this work now? I can't find it in the application, using the premium version.

No, I marked it as duplicated because it's a duplicate of this issue
